tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -325,6 +325,53 @@ def verify_email(self, user):
             )
 
 
+class TokenAuthorizationXMS(TokenAuthorizationOIDC):
+    """Define methods for handling login request from XMS."""
+
+    def decode_payload(self, token_data, options=None):
+        """Decode the payload with keys for XMS."""
+        # TODO: Refactor to XMS login
+        pass
+
+    def get_token_endpoint_response(self, code):
+        """Build out the query string params and full URL path for token endpoint."""
+        # TODO: Refactor to XMS login
+        pass
+
+    def get_auth_options(self, access_token, sub):
+        """Add specific auth properties for the CustomAuthentication handler."""
+        # TODO: Refactor to XMS login
+        pass
+
+    def verify_email(self, user):
+        """Handle user email exception to disallow ACF staff to utilize non-AMS authentication."""
+        # TODO: Refactor to XMS login
+        pass
```

Comment:
```suggestion

```

Not needed for XMS, the inherited function from TokenAuthorizationOIDC should be sufficient.